src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/config/ServiceConfigItem.java

```diff
@@ -25,6 +25,11 @@
      */
     private String transformationUrl;
 
+    /**
+     * URL to an endpoint that handles update existing case upon exception record to a service specific case.
```

Comment:
Worth mentioning that this endpoint is for providing data, not doing actual update. This is a suggestion, up to you.